axlearn/cloud/gcp/jobs/bastion_vm.py

```diff
@@ -317,11 +317,23 @@ def _execute(self):
 class SubmitBastionJob(BaseSubmitBastionJob):
     """A job to submit a command to bastion.
 
+    TODO(rpang): rename this class to BastionRemoteJob.
+
     Main differences from base submit:
     - Emits gsutil commands to view logs.
     - Emits a warning if the bastion doesn't exist in GCE.
     """
 
+    @config_class
+    class Config(BaseSubmitBastionJob.Config):
+        zone: Required[str] = REQUIRED
+
+    @classmethod
+    def from_flags(cls, fv: flags.FlagValues, **kwargs) -> Config:
+        cfg = super().from_flags(fv, **kwargs)
+        cfg.zone = fv.zone
```

Comment:
I think we can remove this (`from_flags` defaults to setting configs that match flag names).

Reply:
Oh, that's neat. Thanks!